src/Nethermind/Nethermind.Evm/VirtualMachine.cs

```diff
@@ -855,7 +859,7 @@ private CallResult ExecuteCall<TTracingInstructions>(EvmState vmState, ReadOnlyM
         ref readonly ExecutionEnvironment env = ref vmState.Env;
         ref readonly TxExecutionContext txCtx = ref env.TxExecutionContext;
         ref readonly BlockExecutionContext blkCtx = ref txCtx.BlockExecutionContext;
-        ReadOnlySpan<byte> code = env.CodeInfo.MachineCode.Span;
+        ICode code = env.CodeInfo.MachineCode;
```

Comment:
this is a really bad for performance.
try to construct a read only span from the available code chunks for verkle